File: content/10/a.dart
main() {
   var s, n = 1;
   switch (n) {
   case 3:
      s = 'all';
      break;
   case 2:
   case 1:
      s = 'some';
      break;
   default:
      s = 'none';
   }
   print(s == 'some');
}

File: content/10/a.php
<?php
$n = 10;

switch ($n) {
case 12:
   $s = 'all';
   break;
case 11:
case 10:
   $s = 'some';
   break;
default:
   $s = 'none';
}

var_dump($s == 'some');
